### PR TITLE
Move to ts

### PR DESCRIPTION
**Pull request by dkildar**

### CODE REVIEW

File: src/types/get-image-entry.interface.ts [hunk 1]
```diff
@@ -0,0 +1,7 @@
+export interface GetImageEntry {
```
[the remark this answers is not in the document]
Comment (dkildar) [reply src/types/get-image-entry.interface.ts:1]: - [ ] Rename and pass to getPostBodySummary and markdown2Html

File: src/helper.ts [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+  if (html.trim() === '') {
+    return null
+  }
+
+  const doc = DOMParser.parseFromString(html, 'text/html')
+
+  return doc
+}
+
+export function domSerializer(): xmldom.XMLSerializer {
```
[the remark this answers is not in the document]
Comment (dkildar) [reply src/helper.ts:14]: - [ ] Check it

[the remark this answers is not in the document]
Comment (dkildar) [reply src/helper.ts:14]: I checked We use xmldom in few methods

File: src/post-body-summary.ts [hunk 3]
```diff
@@ -30,7 +31,7 @@ function postBodySummary(entryBody: string, length?: number): string {
   return text
 }
 
-export function getPostBodySummary(obj: any, length?: number): any {
+export function getPostBodySummary(obj: Entry, length?: number): any {
```
Comment (talhasch) [inline src/post-body-summary.ts]: `obj` type should be `Entry | string` ?

Comment (dkildar) [reply src/post-body-summary.ts]: It’s outdated. It fixed.

**Review by feruzm — APPROVED** [review]
lgtm 🚀

**Pull request merged**